src/components/modals/templates.js

```diff
@@ -0,0 +1,128 @@
+import React, { forwardRef } from 'react';
+import styled from 'styled-components';
+import { useSpring, animated, config } from 'react-spring';
+
+const animations = {
+  fade: [{ opacity: 0 }, { opacity: 1 }],
+  fadeUp: [
+    {
+      opacity: 0,
+      transform: 'scale(0.99) translate3d(0, 10px, 0)'
+    },
+    {
+      opacity: 1,
+      transform: 'scale(1) translate3d(0, 0, 0)'
+    }
+  ]
+};
+
+const Bg = styled(animated.div)`
+  position: fixed;
+  width: 100vw;
+  height: 100vh;
+  top: 0;
+  left: 0;
+  z-index: 10;
+  overflow-y: scroll;
+`;
+
+const SimpleBg = styled(Bg)`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+const FullscreenModal = forwardRef(
+  ({ show, onClose, modalProps, children }, ref) => {
```

Comment:
by the time this gets called, children is null & throws an error on ln58, although in the `ModalProvider` `children` is defined.